tests/unit/amo/pages/TestGuides.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import { fetchGuidesAddons } from 'amo/reducers/guides';
+import {
+  dispatchClientMetadata,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import Guides, { extractId, GuidesBase, getContent } from 'amo/pages/Guides';
+
+describe(__filename, () => {
+  let content;
+  let store;
+  let slug;
+  let i18n;
+  let guids;
+
+  beforeEach(() => {
+    store = dispatchClientMetadata().store;
+    slug = 'privacy';
+    i18n = fakeI18n();
+    content = getContent(slug, i18n);
+    guids = content.sections.map((section) => section.addonGuid);
+  });
+
+  const getProps = ({
+    dispatch = store.dispatch,
+    match = {
+      params: {
+        slug,
+      },
+    },
+    ...customProps
+  } = {}) => {
+    return {
+      dispatch,
+      i18n,
+      match,
+      store,
+      ...customProps,
+    };
+  };
+
+  const render = ({ ...customProps } = {}) => {
+    const allProps = getProps(customProps);
+
+    return shallowUntilTarget(<Guides {...allProps} />, GuidesBase);
+  };
+
+  it('fetches guides addons', () => {
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const root = render({ dispatch: dispatchSpy });
+
+    sinon.assert.calledWith(
+      dispatchSpy,
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: root.instance().props.errorHandler.id,
+      }),
+    );
+  });
+
+  it('renders a Guides Page', () => {
+    const root = render();
+
+    expect(root.find('title')).toHaveText(content.title);
+    expect(root.find('.Guides')).toHaveLength(1);
+    expect(root.find('.Guides-header-icon')).toHaveLength(1);
+    expect(root.find('.Guides-header-page-title')).toHaveLength(1);
+    expect(root.find('.Guides-header-intro')).toHaveLength(1);
```

Comment:
You should probably get some content (`GuideType`) first, then pass that content to the `render()` function along with the slug, and then assert for content strings in each section here. We know that it will be rendered given that the parent component (`.Guides`) is rendered.